README.md

```diff
@@ -51,6 +51,12 @@ Use vim-plug by adding to your `.vimrc` in your plugin section:
 Plug 'editorconfig/editorconfig-vim'
 ```
 
+Source your `.vimrc`:
+
+```viml
+source .vimrc
```

Comment:
What if the current directory isn't the Vim directory?  How about `source $MYVIMRC`?

Reply:
I agree that would be better.